pyccl/covariances.py

```diff
@@ -198,7 +209,7 @@ def sigma2_B_from_mask(cosmo, a=None, mask_wl=None, p_of_k_a=None):
 
     Args:
         cosmo (:class:`~pyccl.core.Cosmology`): a Cosmology object.
-        a (float, array_like or `None`): an array of scale factor
+        a_arr (float, array_like or `None`): an array of scale factor
```

Comment:
We say that it accepts `None`, but if you try it will miserably fail. We should add a few lines to support this option.